go.mod

```diff
@@ -51,8 +51,8 @@ require (
 	github.com/prometheus/client_model v0.3.0 // indirect
 	github.com/prometheus/common v0.37.0 // indirect
 	github.com/prometheus/procfs v0.8.0 // indirect
-	go.uber.org/atomic v1.9.0 // indirect
-	go.uber.org/multierr v1.7.0 // indirect
+	go.uber.org/atomic v1.7.0 // indirect
```

Comment:
this looks stange - we're downgrading dependencies. is it expected?

Reply:
I removed the indirect dependencies and let `go mod tidy` figure out the ones needed...

Reply:
do we need to do that as part of bumping go version?
I updated the go to 1.20 version and then run `go mod tidy`, I can't see any dependencies changed.
Was it a separate step that you took?

Reply:
I've been updating the version in various projects and noticed that sometimes the `// indirect`s seem get out of sync, so I just manually remove the entire `require` block of `// indirect` and let `go mod tidy` regenerate it. 

In theory it should regenerate the same exact thing, maybe I should open an issue on https://github.com/golang/go 🤔

Reply:
I think we should just check-in the update to go and leave out the dependency changes that were introduced by removing the `// indirect`s.  If we run into issues later with out-of-sync `// indirect`s we can open another PR.

Reply:
Sure. I've updated the PR.